### DISCUSSION TITLE
Quarkus 3.8.2 released - Maintenance release

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#39278] [Quarkus Blog/Website]
# Quarkus 3.8.2 released - Maintenance release

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-8-2-released/

**Comment by turing85** [comment]
For those who want to upgrade: pleaes be aware of https://github.com/quarkusio/quarkus/issues/39283
